### DISCUSSION TITLE
Welcome to room-assistant Discussions!

### DISCUSSION THREAD

**Discussion by mKeRix** [mKeRix/room-assistant#333] [General]
## 👋 Welcome!
  This repo is using Discussions as a place to connect with members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by Okosotthon** [comment]
Hello and thanks for your hard work, really appreciated 👍 .
